app/src/androidTest/java/com/github/palFinderTeam/palfinder/MainActivityTest.kt

```diff
@@ -13,25 +13,3 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
-
-@RunWith(AndroidJUnit4::class)
-class MainActivityTest {
-    @get:Rule
-    val testRule = ActivityScenarioRule(MainActivity::class.java)
-
-    @Test
-    fun testSendMessage(){
-        val message = "username"
-        Intents.init()
-        onView(ViewMatchers.withId(R.id.mainName))
-            .perform(ViewActions.replaceText(message))
-            .perform(ViewActions.closeSoftKeyboard())
-
-        onView(ViewMatchers.withId(R.id.mainGoButton))
-            .perform(ViewActions.click())
-
-        intended(IntentMatchers.hasExtra(EXTRA_MESSAGE, message))
-        Intents.release()
-        }
-
-}
```

Comment:
maybe keep a test to check if you went to your activity even if it's the old button

Reply:
I think I am going to do a better test for it when we merge everything together for the demo, but thank you for the comment !